.github/linters/.markdown-lint.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Linter rules doc:
+# - https://github.com/DavidAnson/markdownlint
+
+###############
+# Rules by id #
+###############
+MD004: false                  # Unordered list style
+MD007:
+  indent: 2                   # Unordered list indentation
+MD013:
+  line_length: 400            # Line length 80 is far to short
```

Comment:
"too short"

Reply:
Most of this configuration comes from the default github superlinter configuration file. I have added some checks to ignore.
Line length I left with a default value. 
When I view .md documents on github/IDE everything looks fine. With a line length of 400, we have no problem with formatting the tables.

Reply:
I meant the comment: Line length 80 is **to** short
:)